### DISCUSSION TITLE
WiX Toolset 5.0.0-rc.1 <Files> issue with subfolders

### DISCUSSION THREAD

**Discussion by Thylon1** [wixtoolset/issues#8039] [Questions] [answered]
Hello,
I wanted to test the new Wix5 Files feature but encountered a wierd issue.
Example repo: https://github.com/Thylon1/WebApplication1

I am trying to use the new `<Files>` element to include all files and subfolders and their respective files in my MSI file.
The MSI builds with no errors or warnings, but when I install the MSI on a test system, all the subfolders are nested within themselves.

bin folder:
![grafik](https://github.com/wixtoolset/issues/assets/77726908/07b66851-f4d5-4dfd-8987-e75d7f3a42b3)
Output:
![grafik](https://github.com/wixtoolset/issues/assets/77726908/5c9668de-1afa-43b6-9d91-e7465f0f65b6)

Does the `<Files>` element support subfolders or do I need to specify each subfolder separately?

**Comment by barnson** [comment, marked as answer]
It does support subfolders. Please open a bug. (And thanks for the repro repo.)

**Reply by Thylon1** [reply]
I have opened the issue #8040 
Thank you.
